test/bigtest/network/config.js

```diff
@@ -122,9 +122,17 @@ export default function configure() {
         right,
       } = cqlParser.tree;
 
-      if (left && right && left.field === 'title' && right.field === 'contributors') {
-        return instances.all().filter(inst => inst.title === left.term &&
-          inst.contributors[0].name === right.term);
+      if (left && right) {
+        if (left.field === 'title' && right.field === 'contributors') {
```

Comment:
I wonder if we could use an optional chaining here instead of a nested 'if'? 
```suggestion
        if (left?.field === 'title' && right?.field === 'contributors') {
```

Reply:
@SergiyVSergiyenko nice idea!